setup.py

```diff
@@ -4,5 +4,8 @@
       description='Pipelines for the analysis of the DepMap omics data',
       url='https://github.com/broadinstitute/ccle_processing',
       # install_requires=open("requirements.txt").readlines(),
-      packages=['depmapomics']
+      packages=['src']
```

Comment:
doesn't this cause the package name to become src?

Reply:
@jkobject

Reply:
you are right: I renamed it to depmapomics!